### ISSUE TITLE
Index out of range error in report creation

### ISSUE THREAD

**Issue by adelavega** [poldracklab/fitlins#181]
```
INFO:nipype.workflow:[Job 32] Completed (fitlins_wf.ds_design).
Traceback (most recent call last):
  File "/opt/miniconda-latest/envs/neuro/bin/neuroscout", line 11, in <module>
    load_entry_point('neuroscout-cli', 'console_scripts', 'neuroscout')()
  File "/src/neuroscout/neuroscout_cli/cli.py", line 60, in main
    command(deepcopy(args)).run()
  File "/src/neuroscout/neuroscout_cli/commands/run.py", line 57, in run
    retcode = run_fitlins(fitlins_args)
  File "/opt/miniconda-latest/envs/neuro/lib/python3.6/site-packages/fitlins/cli/run.py", line 215, in run_fitlins
    report_dict = build_report_dict(deriv_dir, work_dir, analysis)
  File "/opt/miniconda-latest/envs/neuro/lib/python3.6/site-packages/fitlins/viz/reports.py", line 83, in build_report_dict
    for contrast in contrasts]
  File "/opt/miniconda-latest/envs/neuro/lib/python3.6/site-packages/fitlins/viz/reports.py", line 83, in <listcomp>
    for contrast in contrasts]
IndexError: list index out of range

```

This error occurs with most models, such as this one in this instance:
https://neuroscout.org/builder/MXKyA

But not in a model with only 1 run per subject: 
https://neuroscout.org/builder/wOlOA

**Comment by adelavega** [comment]
The traceback masks this, but the error is in getting the glassbrain:
```
                     'glassbrain': fl_layout.get(contrast=snake_to_camel(contrast.name),
                                                 suffix='ortho', extension='png', **ents)[0].path}
```

**Comment by effigies** [comment]
Presumably the `**ents` is too specific.

**Comment by adelavega** [comment]
Yeah, that's what I figured: 

```
{'acquisition': '374vol', 'run': 1, 'space': 'MNI152NLin2009cAsym', 'subject': 'rid000001', 'task': 'life'}
```

Seems like all of our problems lately have to do with ents, and the fact that too many unrelated things are getting bunched together.

**Comment by adelavega** [comment]
Looks like this is the target file:
`sub-rid000001_task-life_run-1_space-MNI152NLin2009cAsym_contrast-speech_stat-t_ortho.png`

Maybe acquisition should be in the filename? Obviously the easy fix is to omit that from those `ents`... but they were in the original input filenames.

**Comment by effigies** [comment]
Ah, yeah. I probably just don't have it in the path patterns.

**Comment by adelavega** [comment]
Is it possible two have two acquisitions with the same run id? I guess if you only had one run (no run id) that might be important to disambiguate. 

I pushed a quick change to remove this entities, but we can add to path patterns instead, your call.

**Comment by effigies** [comment]
Yes, run indices only need to be unique when holding all other entities constant. So we should probably make sure we can handle any valid entities for `bold` files.

**Comment by adelavega** [comment]
okay, I will revert that change then, we can take care of it in a separate PR.

**Comment by adelavega** [comment]
Looks like the issue is probably pybids related, as the `BIDSDataSink` for that file doesn't get the `acquisition` entity, while others do:

```
path_pattern = ['reports/[sub-{subject}/][ses-{session}/]figures/[run-{run}/][sub-{subject}_][ses-{session}_]task-{task}[_acq-{acquisition}][_rec-{reconstruction}][_run-{run}][_echo-{echo}][_space-{space}]_contrast-{contrast}_stat-{stat<effect|variance|z|p|t|F>}_ortho.png']
entities = {'contrast': 'speech', 'stat': 't', 'subject': 'rid000001', 'task': 'life'}
out_name = /out/MXKyA/fitlins/reports/sub-rid000001/figures/sub-rid000001_task-life_contrast-speech_stat-t_ortho.png
```

**Comment by adelavega** [comment]
It's not your path_patterns. The `acquisition` entity gets dropped for some reason, even though we're still at the run level:
```
['reports/[sub-{subject}/][ses-{session}/]figures/[run-{run}/][sub-{subject}_][ses-{session}_]task-{task}[_acq-{acquisition}][_rec-{reconstruction}][_run-{run}][_echo-{echo}][_space-{space}]_contrast-{contrast}_stat-{stat<effect|variance|z|p|t|F>}_ortho.png']

{'contrast': 'speech', 'run': 2, 'space': 'MNI152NLin2009cAsym', 'stat': 't', 'subject': 'rid000005', 'task': 'life'}

/out/MXKyA/fitlins/reports/sub-rid000005/figures/run-2/sub-rid000005_task-life_run-2_space-MNI152NLin2009cAsym_contrast-speech_stat-t_ortho.png```

**Comment by effigies** [comment]
It may be the entity whitelist. I'll look for it real quick...

**Comment by effigies** [comment]
@adelavega Do you want to try out #182?

**Comment by adelavega** [comment]
ah yeah, that looks guilty. the l1_model wasn't getting `acquisition` as an entity. 

sure, i'll try it.

**Comment by effigies** [comment]
Is this issue fully addressed, or are there some remaining components?

**Comment by adelavega** [comment]
Fully addressed by #182
